### DISCUSSION TITLE
Welcome to kali-ios Discussions!

### DISCUSSION THREAD

**Discussion by finlandhl** [finlandhl/kali-ios#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Madmaxx10r** [comment]
Hi, 

I'm a beginner with Kali linux. I loved the idea of running Kali on iOS. 

This might be a stupid question but would like to hear your thoughts. Since the app is sandboxed I'm unable to access the device resources. Is there any way to bypass the permission or give access to iSH shell. So I can access the network card and other hardware on the device ?

**Reply by johnschriner** [reply]
Agreed, although it's great to have all of the kali tools available, is there any way to get access to a usb wifi adapter to do anything with wifi tools?  Not easy, I'm sure.  Also there doesn't seem to be any shared directory with iOS; attached drives aren't mountable AFAIK.  It's so great to have pip on my ipad!  Thanks for all of your work on this!

**Comment by Kayada80** [comment]
Bro nothing works

**Comment by Maksiu-K0X** [comment]
It boots to some /19 directory going out of it makes it so u can't go anywhere else git clone command says there is no such directory no metter what I try to clone I am so confused  help🤣

**Comment by W011f** [comment]
Is there any other link other than the MEGA link available?

**Reply by Jaison111** [reply]
https://mega.nz/file/l2EBCb7B#s4hHLfrPogB8dveTt8wI64mRdX2HAEuEZ-1EHRcdCrY

**Comment by ctest4567** [comment]
im confused (first time hacker here) so it says in the requirements to look for a rootfs but i cant find it if i had a gun to my head. can anyone help me?

**Reply by ctest4567** [reply]
guys i figured it out myself im so strong and independent
